OpenOversight/tests/conftest.py

```diff
@@ -563,10 +563,6 @@ def browser(app, request):
     time.sleep(3)
     yield driver
 
-    # shutdown server
-    driver.get("http://localhost:5000/shutdown")
-    time.sleep(3)
-
```

Comment:
removed the special pytest flask invocation from the `browser` fixture (it was starting a new flask with all the defaults, including a default database connection that I couldn't override)